### PR TITLE
i think the problem was that moves list was definied as async

### PR DESCRIPTION
**Pull request by VanditPatel1**
I think you cant define a component as async @preetshah123

### CODE REVIEW

File: src/backend/MoveList.js [hunk 1]
```diff
@@ -5,64 +5,56 @@ import {firestore, root} from "../config/firebase";
...
-		};
-		if (player.data().coins >= 7){
-			setCoup(false)
-		};
-		if (player.data().coins >= 10){
-			setTheRest(true)
-		};
+	useEffect( () => {
+		firestore.collection(root).doc(props.roomName).collection("players").doc(props.activePlayerID).get().then((player)=>{
+			console.log("HEEEEEEEEEEEEEEEEE");
```
Comment (VanditPatel1) [inline src/backend/MoveList.js:16]: lmoa my bad i forgot to take out my debug logs, can you remove them from your brach after the merge

Comment (VanditPatel1) [reply src/backend/MoveList.js:16]: I am definitely NOT trying to improve my LoC

Comment (Shehan29) [reply src/backend/MoveList.js:16]: Sure ... 🙄

Comment (preetshah123) [reply src/backend/MoveList.js:16]: LOL

File: src/backend/MoveList.js [hunk 2]
```diff
@@ -5,64 +5,56 @@ import {firestore, root} from "../config/firebase";
...
 				<div style ={{paddingBottom: "1em", paddingTop: "1em"}}>
 					<Button type="button" className="btn btn-lg btn-light" style = {{width:"20em"}} onClick={all_moves["Steal 2 from a player as Captain"](props.roomName, props.currentTurn, props.playerName, props.activePlayerID)}
-					disabled = {theRest}>Steal 2 from a player as Captain</Button>
+					disabled = {false}>Steal 2 from a player as Captain</Button>
 				</div>
 
 				<div style ={{paddingBottom: "1em", paddingTop: "1em"}}>
 					<Button type="button" className="btn btn-lg btn-light" style = {{width:"20em"}} onClick={all_moves["Assassinate someone you dislike!"](props.roomName, props.currentTurn, props.playerName, props.activePlayerID)}
-					disabled = {assassinEnabled}>Assassinate someone you dislike!</Button>
+					disabled = {false ? coins < 3 : true}>Assassinate someone you dislike!</Button>
```
Comment (Shehan29) [inline src/backend/MoveList.js:51]: Cool use of ternary but can’t you just replace with the inequality?

Comment (VanditPatel1) [reply src/backend/MoveList.js:51]: oh ya, good point, don't need to use ternary there

**Pull request merged**